### PR TITLE
Delete comment from classifiers

### PR DESCRIPTION
**Pull request by almg80**
Causing 
```
  at build/sdk/lib/python3.8/site-packages/poetry/core/packages/package.py:291 in all_classifiers
      287│             classifiers.append(self.license.classifier)
      288│
      289│         classifiers = set(classifiers)
      290│
    → 291│         return sorted(classifiers)
      292│
      293│     @property
      294│     def urls(self):  # type: () -> Dict[str, str]
      295│         urls = {}
```

Error in some local environments and CI

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -16,7 +16,6 @@ keywords = [
     "ML Metadata Store",
 ]
 classifiers = [
-    # As from http://pypi.python.org/pypi?%3Aaction=list_classifiers
```
Comment (dpp23) [inline pyproject.toml:19]: Nit: Can we move it ourside of the list rather than deleting altogether?

**Review by dpp23 — APPROVED** [review]
LGTM if CI passes.

**Review by micheldlebeau — APPROVED** [review]

File: poetry.lock [hunk 2]
```diff
@@ -3204,2260 +3204,226 @@ python-versions = ">=3.7.1,<3.11"
...
-    {file = "wrapt-1.14.1-cp38-cp38-manylinux_2_5_i686.manylinux1_i686.manylinux_2_17_i686.manylinux2014_i686.whl", hash = "sha256:7b7c050ae976e286906dd3f26009e117eb000fb2cf3533398c5ad9ccc86867b1"},
-    {file = "wrapt-1.14.1-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:ef3f72c9666bba2bab70d2a8b79f2c6d2c1a42a7f7e2b0ec83bb2f9e383950af"},
-    {file = "wrapt-1.14.1-cp38-cp38-musllinux_1_1_aarch64.whl", hash = "sha256:01c205616a89d09827986bc4e859bcabd64f5a0662a7fe95e0d359424e0e071b"},
-    {file = "wrapt-1.14.1-cp38-cp38-musllinux_1_1_i686.whl", hash = "sha256:5a0f54ce2c092aaf439813735584b9537cad479575a09892b8352fea5e988dc0"},
-    {file = "wrapt-1.14.1-cp38-cp38-musllinux_1_1_x86_64.whl", hash = "sha256:2cf71233a0ed05ccdabe209c606fe0bac7379fdcf687f39b944420d2a09fdb57"},
-    {file = "wrapt-1.14.1-cp38-cp38-win32.whl", hash = "sha256:aa31fdcc33fef9eb2552cbcbfee7773d5a6792c137b359e82879c101e98584c5"},
-    {file = "wrapt-1.14.1-cp38-cp38-win_amd64.whl", hash = "sha256:d1967f46ea8f2db647c786e78d8c
```
Comment (dpp23) [inline poetry.lock:3271]: I think you used the wrong version of poetry to generate this

Comment (almg80) [reply poetry.lock:3271]: It was due to running it on Mac I think, just finishing a Linux run now to get a good version

Comment (dpp23) [reply poetry.lock:3271]: I think it as the `1.13` version of poetry given they are all empty lists.

**Pull request merged**